utils/addon_uuid.go

```diff
@@ -0,0 +1,52 @@
+package utils
+
+import (
+	"context"
+	"strings"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/go-utils/errors/v2"
+)
+
+func GetAddonUUIDFromType(ctx context.Context, app, addonTypeOrUUID string) (string, error) {
+	// If addon does not contain a UUID, we consider it contains an addon type (e.g. MongoDB)
+	if strings.HasPrefix(addonTypeOrUUID, "ad-") {
```

Comment:
question: Is there a way to verify correctly if it's an UUID?
Because if I pass in argument `ad-toto`, we will do a request to retrieve list of addons for nothing :slightly_frowning_face:

Reply:
The test is already done in db-api side. It would returns an HTTP 404